sklearn/base.py

```diff
@@ -380,7 +387,13 @@ def __setstate__(self, state):
             self.__dict__.update(state)
 
     def __sklearn_tags__(self):
-        return default_tags(self)
+        return Tags(
+            estimator_type=None,
+            target_tags=TargetTags(required=False),
+            transformer_tags=None,
+            regressor_tags=None,
+            classifier_tags=None,
+        )
```

Comment:
Why aren't these the default_tags anymore? Since it's `BaseEstimator`, I assumed that would be "default".

Reply:
cause we need to remove `default_tags` once we're done with the deprecation period. `default_tags` depends on detecting the type of estimator from the class, and not the instance, which we're removing in this PR.